dev/fattest.databases/src/componenttest/topology/database/container/DatabaseContainerType.java

```diff
@@ -12,42 +12,49 @@
 import org.testcontainers.containers.output.OutputFrame;
 
+import com.ibm.websphere.simplicity.config.DataSourceProperties;
 import com.ibm.websphere.simplicity.log.Log;
 
 /**
- * This is a current list of database test-containers that are in the database rotation.
+ * This is a current list of database testcontainers that are in the database rotation.
  */
 @SuppressWarnings("rawtypes")
 public enum DatabaseContainerType {
-    DB2("jcc.jar", "org.testcontainers.containers.", "Db2Container"),
-    Derby("derby.jar", "componenttest.topology.database.container.", "DerbyNoopContainer"),
-    DerbyClient("derbyclient.jar", "componenttest.topology.database.container.", "DerbyClientNoopContainer"),
-    Oracle("ojdbc8_g.jar", "org.testcontainers.containers.", "OracleContainer"),
-    Postgres("postgresql.jar", "org.testcontainers.containers.", "PostgreSQLContainer"),
-    SQLServer("mssql-jdbc.jar", "componenttest.topology.database.container.", "SQLServerContainer");
+    DB2("jcc.jar", "org.testcontainers.containers.", "Db2Container", "Properties_db2_jcc"),
+    Derby("derby.jar", "componenttest.topology.database.container.", "DerbyNoopContainer", "Properties_derby_embedded"),
+    DerbyClient("derbyclient.jar", "componenttest.topology.database.container.", "DerbyClientNoopContainer", "Properties_derby_client"),
+    Oracle("ojdbc8_g.jar", "org.testcontainers.containers.", "OracleContainer", "Properties_oracle"),
+    Postgres("postgresql.jar", "org.testcontainers.containers.", "PostgreSQLContainer", "Properties_postgresql"),
+    SQLServer("mssql-jdbc.jar", "componenttest.topology.database.container.", "SQLServerContainer", "Properties_microsoft_sqlserver");
 
     private String driverName;
-    private Class<? extends JdbcDatabaseContainer> clazz;
+    private Class<DataSourceProperties> dsPropsClass;
+    private Class<? extends JdbcDatabaseContainer> containerClass;
+    
 
     @SuppressWarnings("unchecked")
-	DatabaseContainerType(final String driverName, final String packageName, final String className) {
+	DatabaseContainerType(final String driverName, final String packageName, final String containerClassName, final String dataSourcePropertiesClassName) {
         this.driverName = driverName;
         
-        //Use reflection to get class at runtime.
-        Class clazz = null;
+        //Use reflection to get classes at runtime.
+        Class containerClass = null, dsPropsClass  = null;
 		try {
-			clazz = (Class<? extends JdbcDatabaseContainer>) Class.forName(packageName + className);
+			containerClass = (Class<? extends JdbcDatabaseContainer>) Class.forName(packageName + containerClassName);
+			dsPropsClass = (Class<DataSourceProperties>) Class.forName("com.ibm.websphere.simplicity.config.dsprops." + dataSourcePropertiesClassName);
 		} catch (ClassNotFoundException e) {
-			fail("Could not find class: " + className);
+			fail("Could not find on of the following classes: " + containerClassName + dataSourcePropertiesClassName);
```

Comment:
typo?: "Could not find one..."

Reply:
Nice catch on the typo.  Here I am throwing an error if we fail to find a class during one of these lines: 
```java
containerClass = (Class<? extends JdbcDatabaseContainer>) Class.forName(packageName + containerClassName);
dsPropsClass = (Class<DataSourceProperties>) Class.forName("com.ibm.websphere.simplicity.config.dsprops." + dataSourcePropertiesClassName);
```
Do you think I should try then one at a time and have a more specific failure?